MikuMikuModel/Nodes/Sprites/SpriteSetNode.cs

```diff
@@ -63,6 +63,40 @@ protected override void Initialize()
                     }
                 }
             } );
+            RegisterCustomHandler("Scale all sprites...", () =>
+            {
+                string input = "2.0";
+
+                while (true)
+                {
+                    using (var inputDialog = new InputDialog { WindowTitle = "Scale all sprites", Input = input })
+                    {
+                        if (inputDialog.ShowDialog() != DialogResult.OK)
+                            break;
+
+                        double dfactor;
+
+                        if (Double.TryParse(input, System.Globalization.NumberStyles.AllowDecimalPoint | System.Globalization.NumberStyles.AllowThousands, System.Globalization.CultureInfo.InvariantCulture, out dfactor))
```

Comment:
Why don't you use float's TryParse method instead?
You should also declare the variable in the out statement like so: `out float factor`